### ISSUE TITLE
Unable to add freetext values with `onebox` or `tag` input types when field uses a vocabulary

### ISSUE THREAD

**Issue by alanorth** [DSpace/DSpace#8686]
**Describe the bug**
In the DSpace 4/5/6 XMLUI submission form it was possible to use a onebox input type with an external `<vocabulary>`, which allowed editors to _either_ select values from a popup list or type a value freely:

<p align="center">
  <img width="600" src="https://user-images.githubusercontent.com/191754/221833395-e3414c13-dc60-4b44-8bea-00f406e20380.png" />
</p>

As of DSpace 7.x this behavior is no longer possible with both the `onebox` and `tag` input types:

<p align="center">
  <img width="600" src="https://user-images.githubusercontent.com/191754/221833510-02b989c2-516a-4a14-b2da-4d0fd146e8ff.png" />
</p>

In the screenshot above I have DSpace 7.5 configured with two fields, one a `onebox` and the other a `tag` type, both of which are using the `srsc` vocabulary. I am unable to enter any keywords that don't come from the vocabulary (I tried with comma and semicolon as you can see).

**To Reproduce**
1. Create an input field in your `submission-forms.xml` of type `onebox` or `tag` and [configure it to use an external vocabulary](https://wiki.lyrasis.org/display/DSDOC7x/Submission+User+Interface#SubmissionUserInterface-ConfiguringControlledVocabularies) XML file like `srsc`.
2. Then try to type free text values in the input box in the Angular submission form

**Expected behavior**
It's unclear if this behavior in DSpace 4/5/6 was by design, or just happenstance. In any case, it is implied that, without the `closed=true` attribute the vocabulary. Maybe that's the actual bug here...

**Related Work**
Perhaps related to this issue, ironically submitted by me: https://github.com/DSpace/DSpace/issues/6593

**Comment by tdonohue** [comment]
@alanorth : I agree this is a bug.  Freetext values should be allowed in vocabulary-based fields as long as it does NOT specify `closed=true`.  I.e. `closed=true` means no freetext, while `closed=false` (default) should allow for freetext.

So, it sounds like either freetext is never being allowed, or somehow `closed=true` has become the default.  In either case, I'll pull this over to our 7.6 board and see if we can find a volunteer to dig into it further.

**Comment by alanorth** [comment]
@paulo-graca do you have an idea how to track this down? I can move this issue to the DSpace-angular repository. Would be good to get this sorted in DSpace 7.6, though the timing is tight! I can help review / test if someone comes up with a patch.
